README.md

```diff
@@ -5,24 +5,100 @@ Python bindings for the awesome Openpose library.
 Openpose github page:
 https://github.com/CMU-Perceptual-Computing-Lab/openpose
 
+notice!! PyOpenPose uses OpenCV 3.2 and protobuf 2.6.1
 
-### Building
+If you use other operating systems, you have to make sure the versions of opencv and protobuf are correct.
 
-Clone and build openpose. I assume you will also build caffe inside openpose 3rdparty folder.
-Set an environment variable named OPENPOSE_ROOT pointing to the openpose root folder.
-
-Inside the root folder of PyOpenpose run cmake and build with:
+### Install OpenCV 3.2 in ubuntu 16.04 or 14.04
```

Comment:
opencv3 building instructions should not be covered here. Maybe this level of details should go in a separate file (i.e install.md)